### ISSUE TITLE
Getting wild meat estimates for other countries

### ISSUE THREAD

**Issue by ZoologyDave** [ZoologyDave/WildlifeTradeNutrition#8]
1) We (@ZoologyDave can do this) can dig into additional literature to get country estimates (see below)

2) Someone else have a look at consumption vs pcGDP or some other correlates (any ideas on what?) to look for patterns, and potentially look at regional trends too.

- (D.-M. Cawthorn and L.C. Hoffman. The bushmeat and food security nexus: a global account of the contributions, conundrums and ethical collisions. Food Research International 76, 906- 925 (2015).
 - J.E. Fa, C.A. Peres, J. Meeuweg. Bushmeat exploitation in tropical forests: an intercontinental comparison. Conservation Biology 16, 232-237 (2002)
 - W.J. Ripple et al. Bushmeat hunting and extinction risk to the world’s mammals. Royal Society of Open Science 3,160498 (2016).

**Comment by BollieHooth** [comment]
Re. point number 2 (and might also be some data in here for point number 1)

https://www.sciencedirect.com/science/article/pii/S0006320717305888?casa_token=BaqRcgxbKzMAAAAA:1CeuGSyCBwYIr6_QAP0xy4Jz6yESabo_DuYRTpaVO7v0QVSUm9nEr8SgP9iZXuObGSQ7zePoSPc
![Screen Shot 2020-06-03 at 13 47 13](https://user-images.githubusercontent.com/65966773/83638768-936e0480-a5a1-11ea-9ca3-78bee4d46696.png)

**Comment by BollieHooth** [comment]
Also some stuff here. It's a national case study, but gives some indicators: 

Income or wealth, land cover, distance of village to the nearest park boundary, and level of education of the head of the household were among the factors that significantly related to the likelihood of consuming any of the 10 most commonly consumed species of bushmeat. Household size was the predictor most strongly associated with quantities of bushmeat consumed and was negatively related to consumption. Total bushmeat consumption per adult male equivalent increased as household wealth increased and decreased as distance of villages to park boundaries increased. Bushmeat consumption at the household level was not related to unit values (i.e., price estimates for a good that typically does not have a market value; estimates derived from willingness to sell or trade the good for items of known price) of bushmeat or the price of chicken and fish as potential substitutes. The median consumption of bushmeat at the village level, however, was negatively related to village mean unit values of bushmeat across all species. Our results suggest that a lack of alternative protein sources motivated even the wealthiest among surveyed households to consume bushmeat. 

https://conbio.onlinelibrary.wiley.com/doi/full/10.1111/j.1523-1739.2011.01802.x?casa_token=eSXh6m9kiasAAAAA%3A-HVgzscm9z7iD2nFUopPlYEQ7fV-nPCgKxQQwRd4BrBy41_GZRadllyRln8auPZPcd1s6y2RYJv0Qh7s

**Comment by BollieHooth** [comment]
More stuff: https://conbio.onlinelibrary.wiley.com/doi/full/10.1111/cobi.12441?casa_token=3HJiK85K6X0AAAAA%3AvkUieXVkDFZTAFXsY4qDi68pKl86KSEKGBabB1aesDc7Uuo0PD3TabZi9B-fR-Nu9Fg0II726Iu_MLYn 

We calculated an index of game depletion (GDI) for each market from the sum of the total number of carcasses traded per annum and species, weighted by the intrinsic rate of natural increase (rmax) of each species, divided by individuals traded in a market... High and low GDI were significantly differentiated by human density and human settlements with >3000 inhabitants. Our results provided empirical evidence that human activity is correlated with more depleted bushmeat faunas and can be used as a proxy to determine areas in need of conservation action .

**Comment by ZoologyDave** [comment]
Good sleuthing. I've got the Halpern data and added it to the repo, only issue is that I have no idea what the units are. Have emailed again to ask.

With the refs you've dug out, I think we want to avoid any that don't get at national patterns. Sub-national patterns with distance to market etc. are going to get a bit weird. That Nielson paper looks like it might have data though...

**Comment by ZoologyDave** [comment]
Update: Nielsen is economic data, so I think it's out (plus they are sampling rural households only, which I don't think we can justify as a basis for country-level averages)

**Comment by hommedesmuffins** [comment]
Happy to go with more data, but also putting in a few quick counterpoints for thought before we go super far down that route:


1) What is the time frame if we try to incorporate the data, and how does this relate to the hopeful timeframe for the paper?

2) How much will the additional data add?

3) How much faffing will we need to incorporate the data?

4) Definitely keep all data at the national level even though consumption/insecurity/etc occurs subnationally. We won't have adequate resolution in the drivers to properly analyze subnational trends in wild meat consumption/food insecurity/etc.

________________________________

**Comment by ZoologyDave** [comment]
Yeah, I'm wondering the same here—there'll definitely be faffing and I'm not sure it'll add _that_ much. Plus it does bring in different data methods etc which might be a bit more of a faff... If the Halpern stuff is super easy to bring in, then I suggest we do that. Otherwise, I'm inclined to just stick with what we've got. But your call @BollieHooth as you're leading...

**Comment by BollieHooth** [comment]
Morning! Sorry for the slow reply, I’ve been focusing on a few other things the past few days, but keen to get back in to this now.

Here are my thoughts:

  1.  First choice - Bring the Halpern stuff in if it’s easy
  2.  If it’s not easy, just stick with what we’ve got as the main results… BUT

     *   Add a simple couple of sentences, as Dave had previously suggested, with a bit of extra analysis in the SI, where we say “however we have data gaps, assuming other countries follow similar regional trends, we might expect values X, Y, Z.”. I think as long as we tone down that we’re not saying those are the actual results, but give some indication of what it might be, that would fly?
     *   … On this topic, understood re. not using local data, though the Nielsen paper did use two national-level indicators to look at patterns between countries: inflation rate of consumer price index and corruption perception index. So, I think it could be defensible to interpolate game pppd based on inflation rate or corruption perception index (instead of using continental averages), and then extrapolate by national population. This would be fairly easy, and somewhat more robust than using regional averages. Even so – I don’t think we should report this as our main result, but use it as additional info to say “ours is massive underestimate as we only have 90 countries, so assuming these very simple assumptions (which have some evidence to back them up), here’s another ball park figure, but the real answer is probably something in between or even more” (and stick the extra bit of analysis in the SI)… I also don’t believe a bunch of the zeros in the GENuS model (e.g. Indonesia), so I think even some of the countries that do have data give an underestimate as well. Either way – we’re being very conservative! And if the reviewers don’t like this addition we could take it out and stick with what we’ve got.

What do you think? Looking forward to catching up later, will try to look at the EID’s stuff before then…

Just FYI, here’s the continent means:

[cid:image001.png@01D63A4E.EF054B50]

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by BollieHooth** [comment]
I've dug up national-level corruption perception index and inflation data for us to play with, if we want to.

**Comment by ZoologyDave** [comment]
OK, brain slowing whirring into gear. 

I think I agree with your strategies 1 and 2, above. So let's avoid modelled stuff in the main text but reference some SI stuff for modelled values figures. 

I do worry that if we model bushmeat consumption based on e.g. inflation and corruption, those things are also going to correlated with potential food insecurity and it'll all get a bit circular. I also worry that the range of countries we've got is going to do some funky stuff to the calcs, but happy if someone wants to have a play and then we can look.

**Comment by BollieHooth** [comment]
Yes. Well, let’s try the Halpern data first, and if that fails we look at interpolating.

To clarify – I’m not suggesting we model the unknown data per se, I’m suggesting we use a super simple ‘value transfer’ approach, where we say “Game consumption in Country X (with data) is a reasonable proxy for game consumption in Country Y without data (because they have roughly the same corruption perception index or consumer inflation index), so we’re just going to assign the same(ish) unit value to Country Y as we see in Country X”. Though we may weight or moderate the proxy value by the relative differences in corruption / inflation, and then of course we’d multiply the unit proxy value by the actual population size in Country Y.

This is a legit approach in stuff like economic valuations, where you collect data for one site and use it as a proxy for other similar sites where there’s no data – I used it for a paper we recently published on estimating the total economic value of shark tourism in Indo, and we used PPP by province (amongst other things) to categorise the different sites and assign weighted proxy values from samples sites to non-sampled sites. Other studies have done similar. We didn’t use modelling, and the reviewers were fine with it.

Am I making sense!?

--
Hollie Booth
DPhil Student (NaturalMotion Graduate Scholar)
Interdisciplinary Centre for Conservation Science, University of Oxford

Twitter: @hollieboothie<https://twitter.com/hollieboothie>
Google Scholar: https://bit.ly/2VB1ZxC
ICCS profile: https://www.iccs.org.uk/person/hollie-booth

https://conservationhierarchy.org/

**Comment by ZoologyDave** [comment]
Yeah OK. I think my point is still there though: if the variables we are using to group countries for bushmeat are also going to influence the other things we are interested in then we're not really getting new info? Let's chat about this when we talk though!

**Comment by BollieHooth** [comment]
Yes, in that corruption probably correlates with food insecurity, for example? Let’s discuss later! (and either way, I recon we don’t spend too much time or get too hung up on it – expediting the paper so that it’s timely and at least gives some directional assessment, especially given all the current debates that are happening on this right now (e.g. today at 4pm: https://www.youtube.com/watch?feature=youtu.be&v=lskSHMD4wDw&app=desktop, featuring both EJ and Sue Lieberman from WCS!) is probably most important, as it’s all just a big ball park anyway…)

--
Hollie Booth
DPhil Student (NaturalMotion Graduate Scholar)
Interdisciplinary Centre for Conservation Science, University of Oxford

Twitter: @hollieboothie<https://twitter.com/hollieboothie>
Google Scholar: https://bit.ly/2VB1ZxC
ICCS profile: https://www.iccs.org.uk/person/hollie-booth

https://conservationhierarchy.org/

**Comment by ZoologyDave** [comment]
Got the Halpern data, but it's in wet weight:

"_wet weight as reported by FAO. The one thing that we did not do, which in hindsight (and following that very useful Edwards paper which came out post this being finalised) is convert this all to edible biomass equivalents. Given that fish and meat are reported differently that is something that would have been better. But I seem to remember that that is also because of inconsistencies in how crops are reported too, it is unclear whether all of the product reported in biomass is food_"

@hommedesmuffins, have you got anything to convert to protein? I'm guessing we need dressing percentage (~50%?), and protein percentage (~20%). Does that work for you? Any better numbers I should be using?

**Comment by hommedesmuffins** [comment]
The SI of Poore and Nemecek (2018) has dressing percentages for a few different animals. If possible, might be better to match these with the type of wild meat instead of using a general assumption. If not possible to match them, I'd go with whatever is lowest to make sure we're providing conservative estimates.


The Poore and Nemecek SI also has protein percent for different meat types. If it's not possible to match these with similar types of wild meat (e.g. pig with wild boar, etc), then maybe take the average of these protein estimates?

________________________________

**Comment by ZoologyDave** [comment]
Unfortunately we've just got "wild meat", so can't differentiate. I'll do two: one with lowest dressing percentage and protein, one with the average of both. Suggest we include the most conservative in the main text and say "it could be even more land, see SI". Will crack on unless I hear from you.

**Comment by BollieHooth** [comment]
Sounds good to me - thanks for getting the data and figuring it out (especially on a Saturday). I’m currently just feeling disgruntled because the 20mph gusts of wind have put me off my cycling plans 😭.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ZoologyDave** [comment]
Just an update on this: I'm assuming the Halpern data is dressed, but not processed (in Poore & Nemecek terms, I am using "Retail Weight + Edible Offal / Hot Standard Carcass Weight"—I can double check this with Rich Cottrell, but he's mad busy right now, so don't want to pester him _too_ much. I'm taking these values from Table S5 in P&N's SI.

I am then using the protein % from Table S1 in P&N.

In each case, I am taking the unweighted average across all the domestic livestock, which equates to 70.75% and 18.375%. In total, this means 1kg in Halpern translates to 130g protein.

Squeal if you object.

**Comment by ZoologyDave** [comment]
Update: re-checked Rich's email. He says:

"wet weight as reported by FAO. The one thing that we did not do, which in hindsight (and following that very useful Edwards paper which came out post this being finalised) is convert this all to edible biomass equivalents."

FAO report "[Data are given in terms of dressed carcass weight, excluding offal and slaughter fats.](http://fenixservices.fao.org/faostat/static/documents/QL/QL_methodology_e.pdf)". 

This still includes bone, so I think the calculations above are still good. @hommedesmuffins you're much more clued up on this, but could you just sense-check that? 

There is an issue in that Halpern data excludes offal and fat, whereas P&N report LCAs inclusive of these, but I think we can ignore this—protein content won't vary _that_ much between carcasses with/without offal and fat.

**Comment by hommedesmuffins** [comment]
Yep - those sound great to me. Let me know when you update the land estimates so I can make new biodiv maps (unless you want to do this with the script on Git).

________________________________

**Comment by ZoologyDave** [comment]
Next issue: the GENuS DB has some weird gaps e.g. Gabon, which we now have game data for. I can't assign game proportionally because I don't have the proportions. Suggest I take regional averages of proportions? God this is tedious

**Comment by hommedesmuffins** [comment]
Using FAO data from their food balance sheets might make more sense? GENuS is derived from this (if I remember correctly), and even if not FAO data is widely accepted and used.

________________________________

**Comment by ZoologyDave** [comment]
Ugh, fine. @BollieHooth this ain't going to be ready by lunchtime! Sorry—much more faffing than anticipated

**Comment by ZoologyDave** [comment]
OK, this is getting tedious. Of course. 

So, FAO _production_ data includes a few countries that we are missing (even accounting for Halpern data). FAO _food balance sheet_ data is missing these.

Given that it's specifically game meat we're worried about (potentially with a limited _international_ market, and therefore with most production contributing to domestic food supply*), would it be justifiable to include these production data when we're missing other data? Thinking about it, this is also what Halpern is (production, not supply), so if we don't want the FAO data, we should ditch Halpern.

Thoughts?

*Although there is the whole bushmeat export market thing, so maybe this isn't justifiable

**Comment by ZoologyDave** [comment]
OK, after phone call:
1) Use GENuS where possible
2) For missing countries use Halpern / FAO production data + Imports - Exports

I'll take means from the last three years unless anyone objects

**Comment by hommedesmuffins** [comment]
What year is the GENuS data from? If this is specified, we should use data from the same year in FAO. If not specified, then average from the last 3 years is perfect.

________________________________

**Comment by ZoologyDave** [comment]
Good shout, it's 2011. I thought Gabon was missing data for 2011, but that's for FBS, so doesn't matter for us.
